.gitignore

```diff
@@ -1,17 +1,18 @@
-package-lock.json
+/.DS_Store
+/package-lock.json
```

Comment:
We keep flipflopping on this because it tends to mean we don't get new apostrophe when we test our own boilerplate, but yes, it's totally wrong for other people starting projects to ignore this.